db/post_migrate/20240108022138_change_enable_admin_sidebar_to_group_post_migration.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class ChangeEnableAdminSidebarToGroupPostMigration < ActiveRecord::Migration[7.0]
```

Comment:
Is there a reason we are running the same migration twice?

Reply:
Oh you're right, I can delete the normal one and keep the post migration. Originally I changed the existing setting from bool -> group instead of just making a new one. Will do this :+1: